samples/contrib/local_development_quickstart/Local Development Quickstart.ipynb

```diff
@@ -140,7 +140,7 @@
    "metadata": {},
```

Comment:
<p>Maybe add <code># Same for below</code> for other pipelines defined in this sample?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Thanks for the suggestion, that makes sense